.github/workflows/check-code-validation.yml

```diff
@@ -118,6 +118,8 @@ jobs:
         uses: ./.github/actions/nx-checkout
       - name: "Minimal yarn install"
         uses: ./.github/actions/minimal-yarn-install
+      - name: Check releases.json revisions
+        run: yarn workspace @trezor/connect-common validate-releases.json
```

Comment:
todo: run only if release.json changes:

```
git diff --name-only be519ebadfe30cc73247742ef254256786232793 29a03dfd282a5a192dd94c2c38211c90e7414ede | cat
```

Reply:
`git diff --name-only origin/develop HEAD | cat`

`$BASE_BRANCH_NAME` for `origin/develop`

Reply:
another option is to move it to https://github.com/trezor/trezor-suite/blob/fix/transport-usb-abort/.github/workflows/test-misc.yml#L1

Reply:
we can use changed files action to conditionally run this - https://github.com/dorny/paths-filter

```
- uses: dorny/paths-filter@v3
  id: changeds
  with:
    filters: |
      releases-json:
        - 'packages/connect-common/files/firmware/t1b1/releases.json'

  # run only if some file in 'src' folder was changed
  - if: steps.changes.outputs.releases-json == 'true'
  - name: Check releases.json revisions
    run: yarn workspace @trezor/connect-common validate-releases.json
  ```